docker/linux/android/Dockerfile

```diff
@@ -51,6 +52,8 @@ ARG TOOLS=commandlinetools-linux-6200805_latest.zip
 ARG CMDLINE_URL=https://dl.google.com/android/repository/${TOOLS}
 ENV ANDROID_SDK_ROOT=/root/starboard-toolchains/AndroidSdk/
 
+RUN rm -rf ${ANDROID_SDK_ROOT}
+
 RUN cd /tmp \
   && mkdir -p ${ANDROID_SDK_ROOT} \
```

Comment:
Move the rm -rf to be between `cd` and `mkdir` here, this way we avoid another RUN command that would create an extra container layer.

Reply:
done